packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -78,6 +80,13 @@ class _CodeViewState extends State<CodeView>
     gutterController = verticalController.addAndGet();
     textController = verticalController.addAndGet();
 
+    if (widget.initialPosition != null) {
```

Comment:
This likely also needs to be in didUpdateWidget so that this state object can manage the scroll position if the widget was updated with a new value for `initialPosition`

Reply:
agree we need to do something in didUpdateWidget but we will need to be very careful. You don't want to update the scrollPosition unless widget.initialPosition != oldWidget.initialPosition.